File: Aigilas/Aigilas/States/GameOverState.cs
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using SPX.States;
using SPX.Core;
using Aigilas.Management;

namespace Aigilas.States
{
    public class GameOverState : State
    {
        private readonly Texture2D _menuBase;

        public GameOverState()
        {
            _menuBase = XnaManager.GetGameOverAsset();
        }

        public void Draw()
        {
            var x = (XnaManager.WindowWidth - _menuBase.Bounds.Right)/2;
            var y = (XnaManager.WindowHeight - _menuBase.Bounds.Bottom) / 2;
            XnaManager.Renderer.Draw(_menuBase, new Vector2(x,y), Color.White);
        }

        public void Update()
        {
            if(Input.IsPressed(Commands.Confirm,0,true))
            {
                StateManager.LoadState(new GameplayState());
            }
        }

        public void LoadContent()
        {

        }
    }
}

File: aigilas/src/aigilas/states/GameOverState.java
package aigilas.states;

import aigilas.management.Commands;
import com.badlogic.gdx.graphics.Color;
import sps.bridge.DrawDepth;
import sps.core.Point2;
import sps.core.SpxManager;
import sps.io.Input;
import sps.states.State;
import sps.states.StateManager;

public class GameOverState implements State {
    public GameOverState() {
    }

    @Override
    public void draw() {
        SpxManager.Renderer.drawString("Game Over", SpxManager.getCenter(), Color.WHITE, 1, DrawDepth.ActionText);
        SpxManager.Renderer.drawString("Press SPACEBAR to restart", new Point2(200, 200), Color.WHITE, 1, DrawDepth.ActionText);
    }

    @Override
    public void update() {
        if (Input.isActive(Commands.Confirm, 0, true)) {
            StateManager.loadState(new GameplayState());
        }
    }

    @Override
    public void loadContent() {

    }

    @Override
    public void unload() {

    }
}
